### PR TITLE
[MRG+1] Cleanup visual_tests and disable browser opening

### PR DESCRIPTION
**Pull request by QuLogic**
Intermixing HTML and code makes this file very difficult to read (for me, at least), so change it to use some simple template forms.

Also, don't automatically open the browser on CI because AppVeyor has one installed and it's a waste to open it.

### CODE REVIEW

**Comment by anntzer** [comment]
Perhaps this script should be moved next to `tools/test_triage.py`, and the two named consistently?

**Comment by QuLogic** [comment]
Done.

**Comment by QuLogic** [comment]
Rebased to fix conflicts.

File: tools/make_icons.py [hunk 1]
```diff
@@ -11,7 +11,7 @@
 """
 
 import matplotlib
-matplotlib.use('agg')
+matplotlib.use('agg')  # noqa
```
Comment (NelleV) [inline tools/make_icons.py]: What does the comment mean?

Comment (QuLogic) [inline tools/make_icons.py]: Skip flake8/pep8 issues with this line. (This line causes a warning about imports not at the top of the file.)

**Pull request merged**